File: src/main/java/client/database/data/PendingWorldTransfers.java
package client.database.data;

public class PendingWorldTransfers {
   private int id;

   private int characterId;

   private int from;

   private int to;

   public PendingWorldTransfers(int id, int characterId, int from, int to) {
      this.id = id;
      this.characterId = characterId;
      this.from = from;
      this.to = to;
   }

   public int getId() {
      return id;
   }

   public int getCharacterId() {
      return characterId;
   }

   public int getFrom() {
      return from;
   }

   public int getTo() {
      return to;
   }
}

File: engine/model/src/main/scala/client/database/data/PendingWorldTransfers.scala
package client.database.data

class PendingWorldTransfers(private var _id: Int, private var _characterId: Int, private var _from: Int,
                            private var _to: Int) {
  def id: Int = _id

  def characterId: Int = _characterId

  def from: Int = _from

  def to: Int = _to
}
